spec/presenters/tree_builder_images_spec.rb

```diff
@@ -0,0 +1,48 @@
+describe TreeBuilderImages do
+  context 'TreeBuilderImages' do
+    before do
+      @template_cloud_with_az = FactoryGirl.create(:template_cloud_with_ems)
+
+      role = MiqUserRole.find_by_name("EvmRole-operator")
+      @group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Instances Group")
+      login_as FactoryGirl.create(:user, :userid => 'instances__wilma', :miq_groups => [@group], :settings => {})
+
+      allow(MiqServer).to receive(:my_server) {FactoryGirl.create(:miq_server)}
+
+      @images_tree = TreeBuilderImages.new(:images, :images_tree, {}, nil)
+    end
+    it 'sets tree to have leaf and not lazy' do
+      root_options = @images_tree.send(:tree_init_options, nil)
```

Comment:
Is this a private method?  If so, is it necessary to test?  Generally, you shouldn't need to test private methods, and should instead opt for testing the public interface (which hits this code) when possible.